code/modules/mob/living/carbon/human/death.dm

```diff
@@ -6,14 +6,14 @@
 		return
 
 	death(1)
-	monkeyizing = 1
 	canmove = 0
 	icon = null
 	invisibility = 101
 	default_gib()
 
 //This will get called often at first until custom gibbing events get made up for each species.
 /mob/living/carbon/human/proc/default_gib()
+	monkeyizing = TRUE
```

Comment:
We set this before everything else so that the procs that update the icons can check it and return early.